packages/common/src/editors/__tests__/autocompleterEditor.spec.ts

```diff
@@ -196,30 +196,19 @@ describe('AutocompleterEditor', () => {
       expect(editor.getValue()).toBe('male');
     });
 
-    it('should dispatch a keyboard event and expect "stopImmediatePropagation()" to have been called when using Left Arrow key', () => {
-      const event = new (window.window as any).KeyboardEvent('keydown', { key: 'ArrowLeft', bubbles: true, cancelable: true });
-      const spyEvent = vi.spyOn(event, 'stopImmediatePropagation');
+    ["ArrowLeft", "ArrowRight", "Home", "End"].forEach((key: string) => {
```

Comment:
just another possible suggestion, I almost never used this and it was in fact added by another user but did you know about the [`describe.each`](https://vitest.dev/api/#describe-each)? It's another way of writing the same kind of test looping, see this one for example

https://github.com/ghiscoding/slickgrid-universal/blob/265a04f2a266fc6f9a54d5cd9b994e2039b6d886/packages/odata/src/services/__tests__/grid-odata.service.spec.ts#L1647-L1664

Reply:
I think we talked about that one already. I still fail to see a clear benefit to the simple to read and understand [].forEach but if you prefer the describe.each syntax I can change that

Reply:
ahh I might have forgot, I'm fine with your approach, I just wanted to mention it in case you didn't know but it looks like you already know about it, so all good :)